apps/web/app/api/v2/client/[environmentId]/displays/lib/display.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  TDisplayCreateInputV2,
+  ZDisplayCreateInputV2,
+} from "@/app/api/v2/client/[environmentId]/displays/types/display";
+import { Prisma } from "@prisma/client";
+import { prisma } from "@formbricks/database";
+import { displayCache } from "@formbricks/lib/display/cache";
+import { validateInputs } from "@formbricks/lib/utils/validate";
+import { DatabaseError } from "@formbricks/types/errors";
+import { getContact } from "./contact";
+
+export const createDisplay = async (displayInput: TDisplayCreateInputV2): Promise<{ id: string }> => {
+  validateInputs([displayInput, ZDisplayCreateInputV2]);
+
+  const { environmentId, contactId, surveyId } = displayInput;
+
+  try {
+    let contact: { id: string } | null = null;
+    if (contactId) {
+      contact = await getContact(contactId);
+    }
+
```

Comment:
Please assign the value directly to the variable and then check if the value exists or not